packages/core/src/App/Containers/EffortlessLoginModal/__test__/effortless-login-modal.spec.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { routes } from '@deriv/shared';
+import EffortlessLoginModal from '../effortless-login-modal';
+
+describe('EffortlessLoginModal', () => {
```

Comment:
@yauheni-deriv I leave it to your consideration. We could improve coverage for this small test case:
<img width="1037" alt="Screenshot 2024-03-04 at 16 04 10" src="https://github.com/binary-com/deriv-app/assets/103181650/19f28ba4-cfa2-43d7-9409-713e41844f5b">